src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -282,7 +319,7 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
       } catch (Exception e) {
         versionString = LIBRARY_VERSION
         LOG.warn("Unable to determine Checker Framework version. Assuming default is being used.")
-        LOG.debug(e)
+        LOG.debug('{}', e.toString())
```

Comment:
This would produce an error that is not associated with the message above (the other lines might squeeze in-between).

How about the following?

```groovy
if (LOG.isDebugEnabled()) {
  LOG.warn("Unable to determine Checker Framework version. Assuming default is being used: {}", versionString, e);
} else {
  LOG.warn("Unable to determine Checker Framework version. Assuming default is being used: {}", versionString);
}
```